### PR TITLE
[www] Tutorial: Minor edits for absolute beginners

### PR DESCRIPTION
**Pull request by yisela**
After doing this tutorial as a complete beginner myself, I thought a couple of lines could be made even more newby-friendly.

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ First create the link to the new page.
 To do that, import the `<Link>` component from the `gatsby-link` package that
 was installed along with the starter.
 
-Unlike the normal HTML `<a>` element, our `Link` component uses `to` for
+Unlike the normal HTML `<a>` element, Gatsby's `Link` component uses `to` for
```
Comment (shannonbux) [inline docs/tutorial/part-one/index.md]: I don't know if the Link component is specific to Gatsby. It might be specific just to React? @KyleAMathews?

**Comment by KyleAMathews** [comment]
Deploy preview for *using-glamor* failed.

Built with commit 89eb69409ac852e782c90c2062076f032e40ab09

https://app.netlify.com/sites/using-glamor/deploys/5a7d137a92226a54a40ab59f

File: docs/tutorial/part-one/index.md (hunk 1 above)
Comment (KyleAMathews) [reply docs/tutorial/part-one/index.md]: Nope, it is Gatsby specific so this is definitely a helpful tweak!

**Comment by jlengstorf** [comment]
Hiya @yisela! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**